tools/export.ps1

```diff
@@ -0,0 +1,2 @@
+$env:Logging:LogLevel:Default="Warning"
+resolverclient show --dtmi "dtmi:com:example:DCM_A;1" --registry C:\code\azure-dtdl-repository\ > DCMA_1_DTDL.json
```

Comment:
Uses the old iteration syntax

Reply:
Is this even something we want here? It will fail if users don't have `C:\code\azure-dtdl-repository` folder on their machine.

Reply:
uups, removing